seatunnel-e2e/seatunnel-engine-e2e/connector-seatunnel-e2e-base/src/test/resources/log4j2.properties

```diff
@@ -16,10 +16,10 @@
 # limitations under the License.
 ################################################################################
 
-rootLogger.level = WARN
+rootLogger.level = INFO
 
 logger.zeta.name=org.apache.seatunnel.engine
-logger.zeta.level=WARN
+logger.zeta.level=INFO
```

Comment:
The log file is too much. Please revert it. You can only set single class like `CheckpointCoordinator` log level to `INFO`. Just like https://github.com/apache/seatunnel/blob/a6bd041dccb5e0442131d94c3715fea3a33d8c35/seatunnel-e2e/seatunnel-engine-e2e/connector-seatunnel-e2e-base/src/test/resources/log4j2.properties#L25